internal/hncconfig/validator.go

```diff
@@ -227,7 +228,9 @@ func (a ancestorObjects) add(onm string, ns *forest.Namespace) {
 }
 
 func (v *Validator) InjectConfig(cf *rest.Config) error {
-	mapper, err := apimeta.NewGroupKindMapper(cf)
+	// FIXME: Where do we get the client from?
+	client := &http.Client{Timeout: 180}
```

Comment:
I'm guessing we can't change the signature of injection functions, so I have no idea how to get the `http.Client` at this point.

Reply:
The previous comment doesn't matter, because the inject functions are no longer used according to the release notes (https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.15.0).

My impression is that the inject stuff should be moved to the construction of the validator, but this turned into a skill issue. Where are the validators constructed?